### PR TITLE
first try adding a table of provided options

### PR DESCRIPTION
**Pull request by mirpedrol**
Address comment in https://github.com/nextflow-io/nf-co2footprint/pull/89

image

### CODE REVIEW

File: plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy [hunk 1]
```diff
@@ -183,6 +183,7 @@ class CO2FootprintFactory implements TraceObserverFactory {
          */
         // nm: size of memory available [GB] -> requested memory
         Long memory = trace.get('memory') as Long
+        memory = 1000000
```
Comment (skrakau) [inline plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy]: @mirpedrol where is this coming from? was this for testing maybe?

Comment (mirpedrol) [reply plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy]: oh sorry, it was indeed for testing! I committed it by mistake

Comment (skrakau) [reply plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy]: but the rest can stay at it is? Then I will use it

Comment (skrakau) [inline plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintFactory.groovy]:
```suggestion
```

**Pull request merged**